<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>助手2号</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_Main">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_Home">
       <layout class="QVBoxLayout" name="layout_HomeContainer">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Settings">
       <layout class="QVBoxLayout" name="layout_SettingsContainer"/>
      </widget>
      <widget class="QWidget" name="page_Quiz">
       <property name="styleSheet">
        <string notr="true">
									/* 全局字体设置 */
									QWidget { font-family: &quot;Microsoft YaHei&quot;; font-size: 12pt; }

									/* 选项对齐 */
									QRadioButton, QCheckBox {
									spacing: 10px; /* 按钮和文字的间距 */
									padding: 5px;  /* 整体内边距，增加点击范围 */
									}

									/* 让圆圈/方框永远对齐第一行文字的顶部 */
									QRadioButton::indicator, QCheckBox::indicator {
									width: 20px;
									height: 20px;
									subcontrol-position: top left; /* 强制靠左上 */
									subcontrol-origin: padding;
									margin-top: 3px; /* 根据字体大小下移一点点，对齐文字基线 */
									}

									/* 底部栏样式 */
									#frame_QuizBottom { background-color: #f0f0f0; border-top: 1px solid #e0e0e0; }
								</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_Quiz">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea_Quiz">
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollContent_Quiz">
           <layout class="QVBoxLayout" name="verticalLayout_QuizContent">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>15</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_QuizTop">
              <layout class="QHBoxLayout" name="horizontalLayout_QuizTop">
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnExitQuiz">
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>&lt; 退出</string>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="spacer_QuizTop1">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_ProgressTimer">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="btnProgress">
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="text">
                    <string>进度: 1/10</string>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lbl_ExamTimer">
                   <property name="styleSheet">
                    <string>font-size: 12pt; color: #FF5722; font-weight: bold;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="spacer_QuizTop2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_QuizRight">
                 <item>
                  <widget class="QLabel" name="lbl_QuizInfoRight">
                   <property name="text">
                    <string>错误: 0</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lbl_QuizSource">
                   <property name="styleSheet">
                    <string>color: gray;</string>
                   </property>
                   <property name="text">
                    <string>来源: -</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_QuestionContent">
              <property name="text">
               <string>[单选题] 题目内容...</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_Options">
              <property name="spacing">
               <number>15</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_Quiz">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="frame_QuizBottom">
              <layout class="QVBoxLayout" name="verticalLayout_QuizBottom">
               <property name="spacing">
                <number>15</number>
               </property>
               <property name="leftMargin">
                <number>15</number>
               </property>
               <property name="topMargin">
                <number>15</number>
               </property>
               <property name="rightMargin">
                <number>15</number>
               </property>
               <property name="bottomMargin">
                <number>15</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnSubmitAnswer">
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>提交答案</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_Nav">
                 <property name="spacing">
                  <number>20</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="btnPrevQ">
                   <property name="text">
                    <string>&lt;&lt; 上一题</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnNextQ">
                   <property name="text">
                    <string>下一题 &gt;&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_ExamConfig">
       <layout class="QVBoxLayout" name="layout_ExamConfigContainer">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="page_History">
       <layout class="QVBoxLayout" name="layout_HistoryContainer">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="page_ParserStrategy">
       <layout class="QVBoxLayout" name="layout_ParserStrategyContainer">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="page_PracticeStrategy">
       <layout class="QVBoxLayout" name="layout_PracticeStrategyContainer">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Card">
       <layout class="QVBoxLayout" name="verticalLayout_Card">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_CardHeader">
          <item>
           <widget class="QPushButton" name="btnBackFromCard">
            <property name="text">
             <string>&lt; 返回答题</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_CardTitle">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>答题卡</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer_CardHeader">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_Card">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollContent_Card">
           <layout class="QVBoxLayout" name="verticalLayout_CardContent">
            <item>
             <layout class="QGridLayout" name="gridLayout_CardButtons">
              <property name="spacing">
               <number>5</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_Card">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
